src/tribol/physics/CommonPlane.cpp

```diff
@@ -436,12 +436,13 @@ int ApplyNormal< COMMON_PLANE, PENALTY >( CouplingScheme const * cs )
         }
       } // end for loop over face nodes
 
-      // comment out debug logs; too much output during tests. Keep for easy 
-      // debugging if needed
-      //SLIC_DEBUG("force sum, side 1, pair " << kp << ": " << -dbg_sum_force1 );
-      //SLIC_DEBUG("force sum, side 2, pair " << kp << ": " << dbg_sum_force2 );
-      //SLIC_DEBUG("phi 1 sum: " << phi_sum_1 );
-      //SLIC_DEBUG("phi 2 sum: " << phi_sum_2 );
+      if (logLevel == DEBUG)
+      {
+         SLIC_INFO("force sum, side 1, pair " << kp << ": " << -dbg_sum_force1 );
+         SLIC_INFO("force sum, side 2, pair " << kp << ": " << dbg_sum_force2 );
+         SLIC_INFO("phi 1 sum: " << phi_sum_1 );
+         SLIC_INFO("phi 2 sum: " << phi_sum_2 );
+      }
```

Comment:
I think this should stay commented out for now @srwopschall?  Or potentially removed/moved to common plane tests?

Reply:
Agree, please leave as commented out. How lines like this are handled with be addressed in future PR.